Source/GoBot/ViewDatabase.swift

```diff
@@ -1141,13 +1141,13 @@ class ViewDatabase {
 
     // MARK: pagination
     // returns a pagination proxy for the home (or recent) view
-    func paginated(onlyFollowed: Bool) throws -> (PaginatedKeyValueDataProxy) {
-        let src = try RecentViewKeyValueSource(with: self, onlyFollowed: onlyFollowed)
+    func  paginatedFeed(with feedStrategy: FeedStrategy) throws -> (PaginatedKeyValueDataProxy) {
```

Comment:
There must be something in that line because both of use made the same mistake 🤣 

```suggestion
    func paginatedFeed(with feedStrategy: FeedStrategy) throws -> (PaginatedKeyValueDataProxy) {
```